Makefile.toml

```diff
@@ -369,14 +369,18 @@ script = '''
 
 set -e
 
-rw_path=$(command -v risingwave || true)
-if [[ -z "${rw_path}" ]]; then
-  echo "$(tput setaf 1)$(tput bold)[ERROR]$(tput sgr0) env $(tput setaf 5)USE_SYSTEM_RISINGWAVE$(tput sgr0) is set, but $(tput setaf 5)risingwave$(tput sgr0) is not found in PATH."
-  exit 1
+if [[ "${USE_SYSTEM_RISINGWAVE}" == docker:* ]]; then
+  tag="${USE_SYSTEM_RISINGWAVE#docker:}"
+  echo "$(tput setaf 5)$(tput bold)[Info]$(tput sgr0) env $(tput setaf 5)USE_SYSTEM_RISINGWAVE$(tput sgr0) is set, will use docker image $(tput setaf 5)risingwavelabs/risingwave:${tag}$(tput sgr0)."
```

Comment:
Shall we have a function to output bold colored text?

Reply:
Didn't quite get your point. 🤔

Reply:
```sh
COLOR_MAGENTA=$(tput setaf 5)
COLOR_DEFAULT=$(tput sgr0)
log_info() {
  # blahblah
}

log_info "env ${COLOR_MAGENTA}USE_SYSTEM_RISINGWAVE${COLOR_DEFAULT} is set, will use docker image ${COLOR_MAGENTA}risingwavelabs/risingwave:${tag}${COLOR_DEFAULT}."
```

We can do this by `source` a `log.sh` or use script extension feature provided by cargo make:

```toml
[tasks.base-script]
script.pre = "echo start"
script.main = "echo old"
script.post = "echo end"

[tasks.extended-script]
extend = "base-script"
script.main = "echo new"
```

Reply:
I suggest this because I saw that 1) some other tasks print logs in a slightly different format, 2) things like `$(tput setaf 5)` are not clear at the first glance.

Reply:
Next time for sure!